src/inmanta/plugins.py

```diff
@@ -234,7 +234,11 @@ def __eq__(self, other: object) -> bool:
     int: inmanta_type.Integer(),
     bool: inmanta_type.Bool(),
     dict: inmanta_type.TypedDict(inmanta_type.Type()),
+    Mapping: inmanta_type.TypedDict(inmanta_type.Type()),
+    collections.abc.Mapping: inmanta_type.TypedDict(inmanta_type.Type()),
```

Comment:
Something weird is going on here. `Mapping` should be from `collections.abc`. I think it's good to support both, but then the qualification must be the other way around (or both qualified if you prefer). Because it's very likely that the import will be updated at some point.

Reply:
Same for sequence.